lib/now.mjs

```diff
@@ -18,6 +18,4 @@
  * @private
  * @return {number} The current date timestamp
  */
-export const now = () => {
-  return +new Date();
-};
+export const now = Date.now;
```

Comment:
IMO it would make sense to remove `lib/now.mjs` at this point since it just re-declares browser functionality. Instead, I'd argue for calling `Date.now()` directly when needed. My argument goes two-fold:

* Importing `now()` instead of using `Date.now()` reduces readability of code since I have to look up how `now` is implemented when reading code calling the func
* A module will lead to some overhead when used with common module bundlers (e.g. webpack), negating the bytes saved with declaring it here

Reply:
I agree! However, I don't know if there was a plan to add other functions to `now.mjs` so I kept it.